src/main/java/com/google/api/codegen/transformer/SurfaceNamer.java

```diff
@@ -39,7 +39,9 @@ public SurfaceNamer(NameFormatter languageNamer) {
     super(languageNamer);
   }
 
-  public static final String NOT_IMPLEMENTED = "$ NOT IMPLEMENTED $";
+  public String getNotImplementedString(String feature) {
```

Comment:
I recall in a previous PR that there was some discussion about failing the generation process in cases like this rather than outputting invalid code. Does that apply here? I don't actually have strong feelings about it, I can see a use case for wanting to generate a partial surface while you are building out a new language.